### DISCUSSION TITLE
Strategic Investment in Solarflare (FLARE) for Enhanced CRAB/CKTON Visibility and Liquidity

### DISCUSSION THREAD

**Discussion by sonoferin2000** [ktondao/KtonDAO#26] [General]
Proposal: Strategic Investment in Solarflare (FLARE) for Enhanced CRAB/CKTON Visibility
Summary

This proposal recommends a strategic investment in Solarflare's FLARE token on the Moonbeam network. By staking FLARE to mint veFLARE 
(voting escrow FLARE), the DAO can gain governance influence, list CRAB and CKTON tokens on Solarflare, and establish price feeds on Coingecko. This will enhance the visibility, liquidity,  and adoption of Crab Network assets, addressing a key gap in the ecosystem.

Background
Solarflare is a decentralized exchange (DEX) operating on the Moonbeam network.
Fully diluted market cap of Solarflare is only $56,608.6, making it a highly undervalued asset with significant growth potential.
CRAB and CKTON tokens currently lack price feeds on platforms like Coingecko, limiting their market visibility.
Staking FLARE allows the DAO to mint veFLARE, which provides governance rights to influence token listings and liquidity incentives, as well 
as possibly  implementing our tech on their exchange. They need bridging, we can help.

Rationale
Visibility for CRAB and CKTON
By listing CRAB and CKTON on Solarflare, we can establish price feeds, making these tokens visible on Coingecko and other price tracking sites. This will significantly improve exposure and attract potential investors and traders.

Liquidity and Accessibility
Listing on Solarflare will provide liquidity for CRAB/CKTON pairs, incentivizing market activity and boosting trading volume.
Solarflare’s governance allows us to influence incentives for CRAB/CKTON liquidity pools.

Cost-Effective Market Entry
With Solarflare's low fully diluted market cap, a modest investment can provide a meaningful impact on voting power on their platform, while giving exposure and liquidity of CRAB and CKTON.

Integration with the Moonbeam Ecosystem
A stronger presence on Moonbeam enhances cross-chain growth and positions Crab Network as a more integral player in the Polkadot ecosystem.

Proposed Action Plan
Purchase FLARE
Invest $500–$1,000 in FLARE tokens on the Moonbeam network based on its low fully diluted market cap.

Stake FLARE to mint veFLARE, gaining governance rights to influence Solarflare’s liquidity incentives for CRAB and CKTON.  Will passivly earn additional FLARE simply by staking our veFLARE, and can periodicly extend our FLARE locks for additional veFLARE for added voting power. 
A small investment can give us a large impact. 
List CRAB and CKTON
Propose the listing of CRAB/GLMR and CKTON/GLMR pairs on Solarflare and establish price feeds to track these assets on Coingecko. 

Incentivize Liquidity Pools
Use veFLARE governance rights to propose liquidity incentives for CRAB and CKTON pools, encouraging market participation and liquidity growth.

Monitor and Expand
Track the performance of the liquidity pools, governance actions, and token exposure.
Explore further integration with the Moonbeam and Polkadot ecosystems.

Expected Benefits
Increased Visibility. Listing CRAB and CKTON on Solarflare and tracking them on Coingecko increases visibility, attracting more users and traders.

Stronger Liquidity
Active liquidity pools improve token utility and trading opportunities, enhancing market depth and accessibility.

Governance Power
veFLARE governance gives the DAO influence over Solarflare’s token incentives, ensuring continued support for CRAB and CKTON.

Revenue Opportunities
Increased trading volume in CRAB/CKTON pools generates fees and potential rewards for the DAO. possibly even revenue from bridging services if we can have that implemented on their exchange as well.

Ecosystem Growth
Solarflare’s integration with Moonbeam positions Crab Network to grow within the broader Polkadot ecosystem.

Risk Assessment
Market Volatility
FLARE’s low market cap means higher volatility. We propose a conservative initial investment to minimize exposure to short-term market fluctuations.

Liquidity Risks
Ensuring participation in liquidity pools requires effective incentives. Regular monitoring and adjustments may be needed to keep pools active.

Conclusion
This proposal presents a low-risk, high-reward strategy to improve the visibility, liquidity, and adoption of CRAB and CKTON by listing them on Solarflare, establishing price feeds, and using veFLARE governance to incentivize liquidity. The DAO is recommended to invest $500–$1,000 in FLARE and execute the proposed actions to enhance the Crab Network ecosystem.

* I was confused about which exchange it was that was connected to Crab, it is Solar token, that officially governs that exchange, but they are both connected,and there is talk of merging both exchanges under a single token, Flare token is the better value proposition I would argue, but leave that for others to decide for themselves. it is the most widely distributed of the 2 tokens. a smaller investment would give you a larger percentage of that,when a vote is made on any token merge.

**Comment by 2xJDubs** [comment]
I think this is a very intriguing idea though I see some hurdles that need to be overcome before moving this to a vote.  

- Your proposal needs to take into account what cKTONDAO has in its treasury currently and how much would be spent of that to perform the proposal.  
- Secondly, who would manage the ownership of the veFLARE and pools?  We would need some sort of multi-sig that would manage that and it would be best if the DAO could manage it remotely from Crab chain where our governance is.  This then roles into the last hurdle.  
- Crab Network is a parachain of Kusama and thus has an easy time sending coins/tokens two and from Kusama parachains.  Solarflare is deployed on Moonbeam which is a parachain of Polkadot and not Kusama.  Now the Solarflare team does have a DEX on Moonriver called Solarbeam which has a similar model.  This could be an option for cKTONDAO but I am not sure how much activity is on Solarbeam these days.

I don't want to discount this idea as I think it is a great start.  We just need to talk through more of the logistics around it.

**Reply by sonoferin2000** [reply]
Thanks for your responce. you brought up some good points. Ideally, I would ask that Kton DAO step up for this. We have to earn our keep in this ecosystem as well, and this would show good will. we can contribute to things too.  I do know, thanks to a link that Hackfisher posted as being KTON DAOs treasury, https://explorer.darwinia.network/address/0x08837De0Ae21C270383D9F2de4DB03c7b1314632 that we have enough on hand to cover it. the full amount of $1000 would reduce that by half. but at the same time, would diversify its holdings, with an interest-bearing investment. Ring sitting passively in the treasury just sits there.

**Comment by sonoferin2000** [comment]
Treasury and Funding:
I’m unsure which wallet serves as the cKTONDAO treasury,
but its holdings are certainly insufficient for this investment.
To proceed, we’d need support from KTONDAO or another group.
KTONDAO does have the resources to contribute and, in doing so,
would diversify its treasury.

veFLARE emissions are paid in FLARE tokens, which can be bonded
or sold for additional veFLARE, increasing voting power. Alternatively,
they could simply hold the FLARE tokens. That decision would be
up to them.

Multi-Sig Management:
You raised an excellent point about multi-sig. If SubDAO operates
cross-chain, as I believe it does, we could leverage it for managing
veFLARE governance and liquidity pools across networks.

Polkadot/Kusama and Solarflare/Solarbeam Integration:
Solarflare and Solarbeam are merging their tokens into a single asset.
This presents a unique opportunity. I currently hold nearly 5% of
veFLARE voting power, which could be increased through staking
rewards or by purchasing veFLARE from the market, potentially at
a better rate.

Together, we could play a significant role in shaping this token merger
and its future direction.

Market Context and Strategic Timing:
The Multichain hack severely impacted Solarflare/Solarbeam, but they
are actively seeking a path forward. Now is an ideal time for us to get
involved and help guide this process.

Solar has a higher price, but Flare has wider distribution. Supporting
their merger could position CRAB and CKTON within a stronger,
revitalized ecosystem.

This proposal isn’t just about visibility or liquidity; it positions us as
strategic partners in a broader effort to rebuild and grow the ecosystem.

**Reply by 2xJDubs** [reply]
I have tried a few times to get the Solarflare team interested in using out tech to build crosschain swaps.  Have talked to FireKid and others.  Now may be a good time to work this forward more and help them with crosschain ability.  I will reach out again.  I think what we could look at doing here is make veFlare part of the index token that Hackfisher proposed.

**Reply by sonoferin2000** [reply]
If we simply put veFLARE in the index, we would miss out on staking rewards,and likely not be able to participate in govornance as well, which was the entire point i was trying to make. we would miss out on keeping the initial FLARE it took to mint the veFLARE, Its  like buying kton with ring, versus staking ring to earn kton.  and retaining ownership of the original ring, which we  could periodicly or regularly extend the lock period in order to recieve additional veFLARE.

**Reply by 2xJDubs** [reply]
How would we miss out on the staking and governance portion if veFLARE is part of the index?  This would be received from staking FLARE and it would be held as part of the index multi-sig which could be controlled by the DAO account for voting.  Lots of possibilities.

**Reply by sonoferin2000** [reply]
because it can't be in 2 places at the same time. it's either staked or it's in the index, it can't exist in 2 different places. if its in the index,  you are not earning staking rewards from the veFLARE because that is in the index un-staked.,and you end up selling off the governance token. that's the point i'm trying to make.

**Reply by 2xJDubs** [reply]
Agreed if it is in a liquidity pool it would not be usable.  I will need to go back and read Hackfisher's idea for more clarity.  In the end,  I think working directly with the team maybe a better path forward.  Not saying that we don't pursue this route as well.  Would love to hear other's thoughts as well.

**Comment by sonoferin2000** [comment]
Im seeing it from a perspective of having leverage in the conversation. we can try working directly, Im not opposed to that. Im just pointing out that we could have a sizeable voice in a  governance scenario for a relativly small amount.

**Reply by 2xJDubs** [reply]
Yes but that amount is fairly heft for KtonDAO.  We need to first understand what fund we have available before considering spending them. @boundless-forest where can we see how much the KtonDAO treasury is holding.  I can see how much RING the staking contract has but I don't know how much of that is treasury and how much is unclaimed rewards.  Thanks.

**Reply by sonoferin2000** [reply]
You made a good point about approaching them directly. That might be the best way to pursue this. I will withdraw this from consideration,

**Reply by 2xJDubs** [reply]
I still think this is something to consider.  Just need to understand the available funds.

**Reply by sonoferin2000** [reply]
maybe we can ask partnershipdude if he could contact x0s0l and see if he can include it. perhaps we can incentivize the pool a little to get it out there in the wild. their exchange could use the traffic. win/win.
